src/controllers/controllerpresetfilehandler.cpp

```diff
@@ -198,7 +196,7 @@ QDomDocument ControllerPresetFileHandler::buildRootWithScripts(const ControllerP
     QDomElement scriptFiles = doc.createElement("scriptfiles");
     controller.appendChild(scriptFiles);
 
-    foreach (const ControllerPreset::ScriptFileInfo& script, preset.scripts) {
+    foreach (const ControllerPreset::ScriptFileInfo& script, preset.getScriptFiles()) {
```

Comment:
Replace legacy Qt `foreach (foo, bar)` macro with standard C++ `for (foo : bar)`